### PR TITLE
[v0.16.x] V16 more backports

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
https://github.com/dashpay/dash/pull/3710 had conflicts
https://github.com/dashpay/dash/pull/3701 had conflicts
https://github.com/dashpay/dash/pull/3707 had conflicts

All others had no conflicts

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
@xdustinface I expect you to create a replacement for https://github.com/dashpay/dash/pull/3732/commits/065300715fd0826757b963632169cd3fa20f293f :D

**Comment by xdustinface** [comment]
> @xdustinface I expect you to create a replacement for [0653007](https://github.com/dashpay/dash/commit/065300715fd0826757b963632169cd3fa20f293f) :D

See 2eb91ad423ff93adeb122754f53930304367feee

**Comment by PastaPastaPasta** [comment]
@xdustinface see latest

**Review by xdustinface — APPROVED** [review]
Looks all good, slightly tested ACK

File: src/qt/overviewpage.cpp [hunk 1]
```diff
@@ -475,13 +457,43 @@ void OverviewPage::privateSendStatus(bool fForce)
 
     if(!walletModel) return;
 
-    auto tempWidgets = {ui->labelSubmittedDenomText,
-                        ui->labelSubmittedDenom};
+    bool fIsEnabled = CPrivateSendClientOptions::IsEnabled();
```
Comment (UdjinM6) [inline src/qt/overviewpage.cpp:460]:
```suggestion
    bool fIsEnabled = privateSendClient.fEnablePrivateSend;
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — APPROVED** [review]
ACK

**Comment by PastaPastaPasta** [comment]
(weird kindof self-)utACK https://github.com/dashpay/dash/pull/3732/commits/dbfdf8cb157954ec350edfbc2ccc5a96582ddc1b

**Review by xdustinface — APPROVED** [review]
ACK

**Pull request merged**